.github/workflows/lint.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Set up Python
+      - name: Set up latest Python
```

Comment:
```suggestion
      - name: Set up latest actions/setup-python Python
```

Reply:
Don't think this is necessary since it's IMO obvious we're going to be using actions/setup-python to set up Python. I know you want it to be explicit the latest Python is coming from that action, but it seems overly explicit.

Reply:
It just hurts my OCD as there could be a later stable version of python released, but the action could not be using it ... That is all.